pkg/cobrautils/links.go

```diff
@@ -32,6 +34,8 @@ func FormatLinkWithHandler(linkhandler func(string) string) func(string) string
 }
 
 func SubstituteCommandLinks(desc string, linkformat func(string) string) ([]string, string) {
+	// This is acceptable as the call chain will just output empty.
+	defer panics.HandlePanic()
```

Comment:
Basically, this panic here is to inform the developer, that there is an error its command description. It is only used for CLIs If the CLI does not enable the ignore panic mode, there would be no change in behavior. And this is not enabled by the CLI (I hope).

Silently ignoring this would result in undetected corrupted command documentation.

Reply:
Sounds good. If this is not library code, the panic is acceptable.